src/renderer/components/title-bar/title-bar.component.tsx

```diff
@@ -76,7 +108,7 @@ export default function TitleBar({ template = "index.html" }: { template: AppWin
                 <div id="drag-region" className="grow basis-0 h-full">
                     <div id="window-title" className="pl-1">
                         <span className="text-gray-800 dark:text-gray-100 font-bold text-xs italic">BSManager</span>
-                        {previewVersion && <span className="bg-main-color-1 text-white dark:text-black dark:bg-white rounded-full ml-1 text-[10px] italic px-1 uppercase h-3.5 font-bold">{previewVersion}</span>}
+                        <TitleBarTags ipcService={ipcService} />
```

Comment:
There is a reason to pass the service as a props, instead of using the `useService` hook directly in the component?

Reply:
Ohh yeah, forgot it was a singleton. Might be better to do that then.